packages/orbit-components/.storybook/preview.ts

```diff
@@ -29,6 +29,25 @@ const preview: Preview = {
   decorators: [orbitDecorator],
   parameters: {
     viewport: { viewports },
+    a11y: {
+      element: "#storybook-root",
+      config: {
+        rules: [
+          {
+            id: "aria-required-attr",
+            selector: "*:not(div.text-heading-title0[role='heading'])",
+          },
+          {
+            id: "color-contrast",
```

Comment:
We also need to disable `scrollable-region-focusable` for the code snippet. Check [this story](https://kiwicom-orbit-sarka-sb-a11y-fixes.surge.sh/?path=/story/popover--default), for example.

<img width="1504" alt="image" src="https://github.com/user-attachments/assets/5f3bf982-65c6-4849-9d89-00291e791bda">

Reply:
yes, I had it there in my first version, I need to revert it back,